models/models.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const db  = require("../server/db");
+const Promise = require("bluebird");
+
+class Communication {
+  static findById(id) {
+    return new Promise((fulfill, reject) => {
+      db("comms")
+      .where("commid", id)
+      .limit(1)
+      .then(function (comms) {
+        if (comms.length > 0) {
+          fulfill(comms[0])
+        } else {
+          fulfill()
+        }
+      })
+      .catch(reject)      
+    })
+  }
+}
+
+class Message {
+  static create(messageObject) {
```

Comment:
Was going to pass message attributes as options but it quickly got confusing. We might want to put more thought into a message creation process. This doesn't same a lot of lines of code, but would be easy to add lots of complex validation logic.